transport_nantes/transport_nantes/settings.py

```diff
@@ -13,6 +13,8 @@
 from django.urls import reverse_lazy
 from . import settings_local
 import sys
+import os
+import rollbar
```

Comment:
Perhaps import at bottom just before calling init?

Reply:
Imports traditionnaly are on top of file, why do you want to move it to bottom ?

Reply:
Moved it to bottom anyway, saves an import in dev, yay

Reply:
Yup, that was the only reason. ;-)